bids-validator/utils/files/readNiftiHeader.js

```diff
@@ -51,11 +51,15 @@ function extractNiftiFile(file, callback) {
     } else {
       fs.read(fd, buffer, 0, bytesRead, 0, function () {
         if (file.name.endsWith('.nii')) {
```

Comment:
Since the buffer is read in L52, can we not immediately close it here, rather than after descending in the call stack?

```suggestion
        fs.close(fd)
        if (file.name.endsWith('.nii')) {
```

Reply:
I am not familiar with the codebase enough to give judgement, is there a unittest for this code - I could then blindly accept ;)

Reply:
@nellh Might know off-hand if this will fail, though...